src/PowerShellEditorServices/Services/TextDocument/Handlers/DocumentHighlightHandler.cs

```diff
@@ -45,7 +46,7 @@ protected override DocumentHighlightRegistrationOptions CreateRegistrationOption
                 request.Position.Line + 1,
                 request.Position.Character + 1);
 
-            if (occurrences is null)
+            if (!occurrences.Any())
```

Comment:
I'd examine any use of `Any` to make sure it's not being called on a yield enumerator. Double enumeration can be fine for entirely "pure" yield methods but it's a foot gun I'd like to avoid if we can